example/rails_app/lib/event_handler.rb

```diff
@@ -2,6 +2,6 @@ module EventHandler
   extend self
 
   def handle_event event
-    puts "RailsApp::EventHandler - Consumed event: #{event.name} with payload: #{event.payload}"
+    puts "RailsApp::EventHandler - Consumed event: #{event.metadata[:name]} with payload: #{event.payload}"
```

Comment:
`event.name` did look nice... not sure if it is as "META" as for example the application generating it... to be in the headers.... just saying not sure about this one :smile: